lib/ansible/modules/remote_management/ansible_playbook.py

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Fabien Dupont <fdupont@redhat.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: ansible_playbook
+short_description: Launch Ansible playbook on remote host
+description:
+  - This module launches an Ansible playbook. It is mainly a
+    wrapper for the ansible-playbook command, so the options
+    match the ansible-playbook options.
+author:
+  - Fabien Dupont (@fdupont-redhat)
+version_added: "2.7"
+options:
+  path:
+    description:
+      - Path of the playbook to run
+    required: true
+    aliases: [name]
+  inventory_hosts:
+    description:
+      - Specify inventory as a comma separated host list
+      - Either `inventory_hosts` or `inventory_path` is required.
+  inventory_path:
+    description:
+      - Specify inventory path (file or directory)
+      - Either `inventory_hosts` or `inventory_path` is required.
+  limit:
+    description:
+      - Further limit selected hosts to an additional pattern
+    required: false
+  extra_vars:
+    description:
+      - Set additional variables as key=value
+    required: false
+  extra_vars_files:
+    description:
+      - Set additional variables from files
+    required: false
+  tags:
+    description:
+      - Only run plays and tasks tagged with these values
+    required: false
+  skip_tags:
+    description:
+      - Do not run plays and tasks tagged with these values
+    required: false
+  connection_method:
+    description:
+      - Connection type to use.
+      - List of connection plugins is available using 'ansible-doc -t connection -l'.
+      - Ansible connection plugins documentation is also available at https://docs.ansible.com/ansible/2.5/plugins/connection.html
```

Comment:
use /latest/ instead of version specific, since we will eventually archive docs for older versions

Reply:
Thanks. Done.